frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -263,3 +268,135 @@ it("clicking on the button only adds the email once", () => {
   expect(handleClose).toHaveBeenCalledTimes(1);
   expect(addEmail).toHaveBeenCalledTimes(1);
 });
+
+describe("edit action dialog helper file", () => {
+  // ============================== VARIABLES ==============================
+  // empty variables
+  const emptyEmailType = "";
+  const emptyEmailTo = "";
+  const emptyEmailCc = "";
+  const emptyEmailResponse = "";
+  const emptyReasonsForActionContent = "";
+  const emptyTaskContent = "";
+
+  // initial variables (for edit only)
+  const initialEmailType = EMAIL_TYPE.replyAll;
+  const initialEmailTo = ["to"];
```

Comment:
In other places, these would be an array of numbers. However, this technically does not cause any issues since it does not actually use any of the defined shapes, so doesn't really matter here.

Reply:
As far as this is an array, the comparison with the initial values and the current values is the same. I'm using the following condition to validate the these values are the same:

`initialEmailToArray.toString() !== emailToArray.toString()`
`initialEmailCcArray.toString() !== emailCcArray.toString()`

The important part here is that it's testing the arrays to make sure that the initial and current values are the same or are different.

Is that making sense?

Reply:
Oh, makes sense. Just a comment. Its all the same in the end here.